app/src/main/res/values-cs/strings.xml

```diff
@@ -231,7 +231,8 @@
     <string name="efgs_title">Spolupráce se zahraničím</string>
     <string name="efgs_header">Pomozte v boji s COVID-19 i při cestách do zahraničí</string>
     <string name="efgs_boundaries">COVID-19 nezná hranice a díky spolupráci mezi evropskými státy vás eRouška může upozornit na riziková setkání i v případě, že jste se potkali s cizinci, kteří používají podobné aplikace jiných států.</string>
-    <string name="efgs_visit">Povolte upozornění na zahraniční riziková setkání, pokud jste byli v posledních 14 dnech v zahraničí nebo tam jezdíte pravidelně.\n\nNastavení můžete kdykoliv změnit v aplikaci.</string>
+    <string name="efgs_visit_onboard">Povolte upozornění na zahraniční riziková setkání, pokud jste byli v posledních 14 dnech v zahraničí nebo tam jezdíte pravidelně.\n\nNastavení můžete kdykoliv změnit v aplikaci.</string>
+    <string name="efgs_visit">Povolte upozornění na zahraniční riziková setkání, pokud jste byli v posledních 14 dnech v zahraničí nebo tam jezdíte pravidelně.</string>
```

Comment:
Teď se ten string `efgs_visit_onboard` duplikuje v tom druhém. Rozděl to prosím na dva stringy a na onboardingu dej 2 stringy za sebe.